services/provider/server/server.go

```diff
@@ -831,6 +844,94 @@ func (s *OCSProviderServer) appendVolumeGroupSnapshotClassExternalResources(
 	return extR, nil
 }
 
+func (s *OCSProviderServer) appendVolumeReplicationClassExternalResources(
+	ctx context.Context,
+	extR []*pb.ExternalResource,
+	consumer *ocsv1alpha1.StorageConsumer,
+	consumerConfig util.StorageConsumerResources,
+	storageCluster *ocsv1.StorageCluster,
+	drRbdStorageId string,
+) ([]*pb.ExternalResource, error) {
+
+	if mirrorEnabled, err := s.isConsumerMirrorEnabled(ctx, consumer); err != nil {
+		return nil, err
+	} else if !mirrorEnabled {
+		return extR, nil
```

Comment:
Let's add a log message saying that we are skipping distribution of vrcs because mirroring is not enabled for the consumer